modules/manage/partials/tiered-storage.adoc

```diff
@@ -1133,6 +1133,76 @@ rpk topic alter-config <topic_name> --set redpanda.remote.read=true
 
 See also: xref:{topic-recovery-link}[Topic Recovery], xref:manage:kubernetes/k-remote-read-replicas.adoc[Remote Read Replicas]
 
+== Pause and resume uploads
+
+IMPORTANT: Redpanda strongly recommends using pause and resume only under the guidance of Redpanda support or a member of your customer success team. 
+
+Starting in version 25.1 and later, when running Tiered Storage you can troubleshoot and resolve temporary issues with a cluster's interaction with cloud storage by pausing and resuming uploads without risking data consistency or loss. To pause or resume segment uploads to cloud storage, use the xref:reference:properties/object-storage-properties.adoc#cloud_storage_enable_segment_uploads[`cloud_storage_enable_segment_uploads`] configuration property (default is `true`), which allows segment uploads to proceed normally after the pause completes and uploads resume.
```

Comment:
```suggestion
Starting in version 25.1, you can troubleshoot issues your cluster has interacting with object storage by pausing and resuming uploads. You can do this with no risk of data consistency or data loss. To pause or resume segment uploads to object storage, use the xref:reference:properties/object-storage-properties.adoc#cloud_storage_enable_segment_uploads[`cloud_storage_enable_segment_uploads`] configuration property (default is `true`). This allows segment uploads to proceed after the pause completes and uploads resume.
```
@Feediver1 I'm going to suggest changing all "cloud storage" to "object storage" to be consistent with the rest of the page!